components/definitions/rewards.d.ts

```diff
@@ -82,6 +83,36 @@ declare namespace Rewards {
     actions: any
   }
 
+  export interface MonthlyReport {
+    month: number
+    year: number
+    balance?: BalanceReport
+    transactions?: TransactionReport[]
+    contribution?: ContributionReport[]
+  }
+
+  export enum ReportType {
+    GRANT = 0,
```

Comment:
for consistency, this should be `GRANT_AD`, however, would read better as `UGP_GRANTS`

Reply:
this is not new enum it's just copied from ledger

Reply:
This was changed from `GRANT` to  `GRANT_UPG` below?

Reply:
synced with ledger